man-roxygen/readctl_vars.R

```diff
@@ -1,6 +1,6 @@
 
-#' @param verbose Should there be verbose output while running the file?
-#' Defaults to FALSE.
+#' @param verbose A logical value specifying if output should be printed
```

Comment:
This behavior is slightly weird to me to have verbose in another template file when it has its own template file.

Reply:
I tried using @template verbose within the template, but it didn't work. I can just remove verbose from this readctl_vars template, though, if that seems more intuitive!

Reply:
I removed it from the template